src/js/components/form-elements/TableBodyVirtualized.jsx

```diff
@@ -117,15 +118,17 @@ class TableBodyVirtualized extends Component {
 
   render() {
     // eslint-disable-next-line max-len
-    const { properties } = this.props;
+    const { properties, minimumBatchSize } = this.props;
     const { totalCount, loadMoreRows, isRowLoaded } = properties;
 
     return (
       <div>
         <InfiniteLoader
           loadMoreRows={loadMoreRows}
           isRowLoaded={isRowLoaded}
-          rowCount={totalCount || 50}
+          rowCount={totalCount}
+          minimumBatchSize={minimumBatchSize}
+          threshold={30}
```

Comment:
@DominikaHatala Perhaps this also could be configurable in config file?

Reply:
It's a good idea to have minimum batch size configurable(now it's set to 50 but maybe we will think that 30 or 100 could be better) but I think there is no sense to make threshold configurable, this property should be constant.